### PR TITLE
refine 13 en docs

### PR DESCRIPTION
**Pull request by shippingwang**
WIP: mse_loss maybe need to create a new sub-title, skip it now.

### CODE REVIEW

**Comment by shippingwang** [comment]
image
image
image
image
image

**Comment by shippingwang** [comment]
image
image
image

**Comment by shippingwang** [comment]
image
image
image
image
image
image
image
image

---
image
image
image
image
image

**Comment by shippingwang** [comment]
image

File: python/paddle/fluid/layers/detection.py [hunk 1]
```diff
@@ -1638,67 +1638,102 @@ def prior_box(input,
...
     sequence according to the aspect_ratios.
 
-    Args:
-       input(Variable): The Input Variables, the format is NCHW.
-       image(Variable): The input image data of PriorBoxOp,
-            the layout is NCHW.
-       min_sizes(list|tuple|float value): min sizes of generated prior boxes.
-       max_sizes(list|tuple|None): max sizes of generated prior boxes.
+    Parameters:
+       input(Variable): 4-D tenosr(NCHW), the data type should be float32 or float64.
```
Comment (heavengate) [inline python/paddle/fluid/layers/detection.py:1651]: NCHW  -> in shape [N, C, H, W], N is the batch size ........

File: python/paddle/fluid/layers/detection.py [hunk 2]
```diff
@@ -1638,67 +1638,102 @@ def prior_box(input,
...
 
         boxes: the output prior boxes of PriorBox.
-        The layout is [H, W, num_priors, 4].
-        H is the height of input, W is the width of input,
-        num_priors is the total
-        box count of each position of input.
+	    4-D tensor, the layout is [H, W, num_priors, 4].
+            H is the height of input, W is the width of input,
+            num_priors is the total
+            box count of each position of input.
```
Comment (heavengate) [inline python/paddle/fluid/layers/detection.py]: 说明一下数据类型

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -6287,27 +6305,63 @@ def l2_normalize(x, axis, epsilon=1e-12, name=None):
...
         epsilon(float): The epsilon value is used to avoid division by zero, \
             the default value is 1e-12.
-        name(str|None): A name for this layer(optional). If set None, the layer \
-            will be named automatically.
-
+	name(str, optional): The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name`
+    
     Returns:
-        Variable: The output tensor variable is the same shape with `x`.
+        Variable: The output has the same shape with `x`.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: 数据类型也说明一下

File: python/paddle/fluid/layers/nn.py [hunk 4]
```diff
@@ -6551,8 +6605,8 @@ def edit_distance(input,
     the edit distance will be divided by the length of reference string.
 
     Args:
-        input(Variable): The indices for hypothesis strings, it should have rank 2 and dtype int64.
-        label(Variable): The indices for reference strings, it should have rank 2 and dtype int64.
+        input(Variable): The indices for hypothesis strings, it should have rank 2 and its data type should be int64.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: have rank 2 ？建议描述得再清晰一下

File: python/paddle/fluid/layers/nn.py [hunk 5]
```diff
@@ -9482,8 +9569,8 @@ def resize_bilinear(input,
               H_out = H_{in} * scale_{factor}
               W_out = W_{in} * scale_{factor}
 
-    Args:
-        input(${x_type}): 4-D Tensor, its data type is float32, float64, or uint8,
+    Parameters:
+        input(Variable): 4-D Tensor, its data type is float32, float64, or uint8,
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: 说明下shape

File: python/paddle/fluid/layers/nn.py [hunk 6]
```diff
@@ -9511,40 +9597,72 @@ def resize_bilinear(input,
         align_mode(bool): ${align_mode_comment}
         data_format(str, optional): NCHW(num_batches, channels, height, width) or 
                                     NHWC(num_batches, height, width, channels). Default: 'NCHW'.
+        name(str, optional): The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name`
 
     Returns:
-        A 4-D Tensor in shape of (num_batches, channels, out_h, out_w) or
-        (num_batches, out_h, out_w, channels).
-
+	4-D tensor(NCHW or NHWC).
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: Variable: 4-D tensor ...

File: python/paddle/fluid/layers/nn.py [hunk 7]
```diff
@@ -6561,27 +6615,29 @@ def edit_distance(input,
         label_length(Variable): The length for each sequence in `label` if it's of Tensor type, it should have shape `[batch_size]` and dtype int64.
 
     Returns:
-        edit_distance_out(Variable): edit distance result in shape [batch_size, 1]. \n
+        edit_distance_out(Variable): edit distance result in shape [batch_size, 1].
         sequence_num(Variable): sequence number in shape [].
         
 
+    Return type: Tuple.
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py]: 不用自己写`Return type`, 请和其它op保持一致。

File: python/paddle/fluid/layers/nn.py [hunk 8]
```diff
@@ -9647,38 +9762,71 @@ def resize_trilinear(input,
                                     Default: 'NCDHW'.
 
     Returns:
-        A 5-D Tensor in shape of (num_batches, channels, out_d, out_h, out_w) or 
-        (num_batches, out_d, out_h, out_w, channels).
+        A 5-D Tensor(NCDHW or NDHWC)
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py]: 这里没写类型

File: python/paddle/fluid/layers/nn.py [hunk 9]
```diff
@@ -9769,38 +9914,71 @@ def resize_nearest(input,
                                     Default: 'NCHW'.
 
     Returns:
-        A 4-D Tensor in shape of (num_batches, channels, out_h, out_w) or 
-        (num_batches, out_h, out_w, channels).
+	4-D tensor(NCHW or NHWC).
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py]: 这里没写类型

File: python/paddle/fluid/layers/nn.py [hunk 10]
```diff
@@ -12050,27 +12225,28 @@ def gaussian_random(shape, mean=0.0, std=1.0, seed=0, dtype='float32'):
...
+        min (Float): minimum , default 0.0.
+        max (Float): maximum, default 1.0.
+        seed (Float): Random seed, default 0. if seed is not 0, will generate same number every time. 
         dtype(np.dtype|core.VarDesc.VarType|str): The type of output data : float32, float_16, int etc
 
     Returns:
-        out (Variable): ${out_comment}
+        sampling tensor.
+
+    Return type: Variable
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py]: 不用自己写Return type， 建议和其它op保持一致

File: python/paddle/fluid/layers/nn.py [hunk 11]
```diff
@@ -15041,29 +15215,18 @@ def npair_loss(anchor, positive, labels, l2_reg=0.002):
...
-    
-    Args:
+    Parameters:
 
-        x(Variable): The input tensor variable.
-        upscale_factor(int): factor to increase spatial resolution
+        x(Variable): 4-D tensor, the data type should be float32 or float64.
+        upscale_factor(int): factor to increase spatial resolution.
 
     Returns:
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py:15848]: 没有返回值，要显示的写None

File: python/paddle/fluid/layers/nn.py (hunk 10 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 9 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 8 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 7 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 6 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 5 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/detection.py (hunk 1 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/detection.py:1651]: hmm...I think it's too redundancy, so I remove the explanation

File: python/paddle/fluid/layers/detection.py (hunk 2 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/detection.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 3 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 4 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py]: OK

File: python/paddle/fluid/layers/nn.py (hunk 11 above)
Comment (shippingwang) [reply python/paddle/fluid/layers/nn.py:15848]: 这里很奇怪，好像上传丢了几行

**Review by wanghaoshuang — DISMISSED** [review]

**Review by wanghaoshuang — DISMISSED** [review]

**Review by heavengate — DISMISSED** [review]

**Review by heavengate — DISMISSED** [review]

**Review by wanghaoshuang — DISMISSED** [review]

**Review by heavengate — DISMISSED** [review]

**Review by wanghaoshuang — DISMISSED** [review]

**Review by heavengate — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**